iron/distribution.yaml

```diff
@@ -6225,6 +6225,22 @@ repositories:
       url: https://github.com/ros-visualization/rqt_topic.git
       version: iron
     status: maintained
+  rrt_planner:
+    doc:
+      type: git
+      url: https://github.com/daviddorf2023/rrt_planner.git
+      version: iron
+    release:
+      tags:
+        release: release/iron/{package}/{version}
+      url: https://github.com/daviddorf2023/rrt_planner.git
+      version: 1.0.0
```

Comment:
You haven't setup a release repository, or done a proper bloom-release yet, so this section shouldn't exist:

```suggestion
```